data_steward/resource_files/fields/person_ext.json

```
@@ -0,0 +1,44 @@
+[
+    {
+        "type": "integer", 
+        "name": "person_id", 
+        "mode": "required", 
+        "description": "A unique identifier for each person."
+    },
+    {
+        "type": "integer",
+        "name": "src_id",
+        "mode": "nullable",
+        "description": "A foreign key to the participant's source EHR site."
```

This should be "string" since it can be like `EHR Site 123`
```suggestion
        "type": "string",
        "name": "src_id",
        "mode": "nullable",
        "description": "A foreign key to the participant's source EHR site."
```